noir-projects/noir-protocol-circuits/crates/blob/src/blob.nr

```diff
@@ -692,5 +727,17 @@ mod tests {
         let d_inversed = D.__invmod();
         assert_eq(d_inversed, D_INV);
     }
+
+    #[test]
+    unconstrained fn compute_sum_and_compute_partial_sums_agree(fields: [Field; FIELDS_PER_BLOB]) {
+        let fields = fields.map(|f| field_to_bignum(f));
```

Comment:
AFAICT, this only tests the "lower end" of the limbs in the `BigNum`: is there an easy way to modify it to cover larger `BigNum`'s, e.g. along the lines of:

```noir
let mut f = field_to_bignum(f);
f.limbs = f.limbs.rev();
..
```

?

Reply:
Field are constructed in the same way in production so this is fine.